### PR TITLE
5.1

### PR DESCRIPTION
**Pull request by cjwest**
@boznik,

Here's the 5.1 deployer

### CODE REVIEW

**Comment by jbickar** [comment]
Can we wait to tag this until we get the `stable20160913` contrib updates in?

**Comment by cjwest** [comment]
@jbickar,
This has all been tagged and is ready to roll.

File: development/product/jumpstart-engineering/jumpstart-engineering.make [hunk 1]
```diff
@@ -1,12 +1,12 @@
 core = 7.x
 api = 2
 
-projects[drupal][version] = "7.43"
+projects[drupal][version] = "7.44"
```
Comment (sherakama) [inline development/product/jumpstart-engineering/jumpstart-engineering.make]: This can go to 50 now.

**Comment by boznik** [comment]
@jbickar , i was under the impression you wanted the contrib modules in this PR to match what is in the stable20160913 branch ?

**Comment by jbickar** [comment]
@boznik yeah but only JSE has been updated in this PR. I'm fine with merging the PR, I'm just saying wait to create a tag the deployer until later this week when `stable20160913` gets merged as well.

File: development/product/jumpstart-engineering/contrib.make [hunk 2]
```diff
@@ -59,15 +59,15 @@ projects[contextual_view_modes][version] = "3.0"
 projects[css_injector][subdir] = "contrib"
 projects[css_injector][version] = "1.10"
 projects[ctools][subdir] = "contrib"
-projects[ctools][version] = "1.9"
+projects[ctools][version] = "1.10"
 projects[custom_breadcrumbs][subdir] = "contrib"
-projects[custom_breadcrumbs][version] = "1.0-alpha1"
+projects[custom_breadcrumbs][version] = "2.0-beta1"
```
Comment (jbickar) [inline development/product/jumpstart-engineering/contrib.make]: This should stay at `1.0-alpha1`

File: development/product/jumpstart-engineering/contrib.make [hunk 3]
```diff
@@ -59,15 +59,15 @@ projects[contextual_view_modes][version] = "3.0"
...
 projects[default_image_ft][subdir] = "contrib"
 projects[default_image_ft][version] = "1.6"
 projects[date][subdir] = "contrib"
 projects[date][version] = "2.9"
 projects[date_ical][subdir] = "contrib"
-projects[date_ical][version] = "2.14"
+projects[date_ical][version] = "3.9"
```
Comment (jbickar) [inline development/product/jumpstart-engineering/contrib.make]: This should stay at `2.14`

File: development/product/jumpstart-engineering/contrib.make [hunk 4]
```diff
@@ -105,9 +105,9 @@ projects[file_entity][version] = "2.0-beta3"
 projects[filefield_paths][subdir] = "contrib"
 projects[filefield_paths][version] = "1.0"
 projects[flag][subdir] = "contrib"
-projects[flag][version] = "3.7"
+projects[flag][version] = "3.8"
 projects[geofield][subdir] = "contrib"
-projects[geofield][version] = "1.2"
+projects[geofield][version] = "2.3"
```
Comment (jbickar) [inline development/product/jumpstart-engineering/contrib.make]: This should stay at `1.2`

**Review by jbickar — COMMENTED** [review]
Plz to clean up major version jumps and match what is on the stack.

**Comment by jbickar** [comment]
See https://github.com/SU-SWS/stanford-jumpstart-deployer/pull/90 for which major version jumps to revert.

File: development/product/jumpstart-engineering/contrib.make [hunk 5]
```diff
@@ -179,7 +179,7 @@ projects[token][version] = "1.6"
 projects[transliteration][subdir] = "contrib"
 projects[transliteration][version] = "3.2"
 projects[uuid][subdir] = "contrib"
-projects[uuid][version] = "1.0-beta1"
+projects[uuid][version] = "1.0-beta2"
```
Comment (boznik) [inline development/product/jumpstart-engineering/contrib.make:182]: 1.0-beta2 has a dependency on php 5.5:
http://cgit.drupalcode.org/uuid/commit/?id=f7b7549
letting go through as only a a test.

File: development/product/jumpstart-engineering/contrib.make [hunk 6]
```diff
@@ -145,13 +145,13 @@ projects[node_convert][version] = "1.2"
...
 projects[nodeformcols][version] = "1.0"
 projects[openlayers][subdir] = "contrib"
-projects[openlayers][version] = "2.0-beta11"
+projects[openlayers][version] = "3.1"
 projects[path_redirect_import][subdir] = "contrib"
 projects[path_redirect_import][version] = "1.0-rc4"
 projects[pathauto][subdir] = "contrib"
 projects[pathauto][version] = "1.3"
 projects[pathologic][subdir] = "contrib"
-projects[pathologic][version] = "2.12"
```
Comment (boznik) [inline development/product/jumpstart-engineering/contrib.make:154]: this should stay on 2.12

File: production/product/jumpstart-engineering/contrib.make [hunk 7]
```diff
@@ -145,13 +145,13 @@ projects[node_convert][version] = "1.2"
...
 projects[openlayers][subdir] = "contrib"
-projects[openlayers][version] = "2.0-beta11"
+projects[openlayers][version] = "3.1"
 projects[path_redirect_import][subdir] = "contrib"
 projects[path_redirect_import][version] = "1.0-rc4"
 projects[pathauto][subdir] = "contrib"
 projects[pathauto][version] = "1.3"
 projects[pathologic][subdir] = "contrib"
-projects[pathologic][version] = "2.12"
+projects[pathologic][version] = "3.1"
```
Comment (boznik) [inline production/product/jumpstart-engineering/contrib.make]: this should stay on 2.12

File: development/product/jumpstart-engineering/contrib.make [hunk 8]
```diff
@@ -145,13 +145,13 @@ projects[node_convert][version] = "1.2"
 projects[nodeformcols][subdir] = "contrib"
 projects[nodeformcols][version] = "1.0"
 projects[openlayers][subdir] = "contrib"
-projects[openlayers][version] = "2.0-beta11"
+projects[openlayers][version] = "3.1"
```
Comment (boznik) [inline development/product/jumpstart-engineering/contrib.make]: this should stay at 2.0-beta11

**Comment by boznik** [comment]
run on deploy. same results for prod and dev:
--- Failed scenarios:

```
/Users/ggarvey/www/linky_clicky/includes/features/jsa-content/page_not_found.feature:7
features/jse_event_series.feature:31
features/jse_get_help.feature:7
```

100 scenarios (97 passed, 3 failed)
472 steps (460 passed, 3 failed, 9 skipped)
16m16.31s (17.67Mb)

**Pull request merged**